### ISSUE TITLE
Why is "@babel/preset-env" not included in package.json?

### ISSUE THREAD

**Issue by Sticksword** [ef4/ember-auto-import#231]
I'm getting an error running `ember serve` after installing `ember-auto-import`:
```
Cannot find module '@babel/preset-env'
```
Upon further inspection, it seems this file is responsible for loading that babel module (https://github.com/ef4/ember-auto-import/blob/master/packages/ember-auto-import/ts/webpack.ts#L146). The problem is, this module is not part of the package.json and so I'm missing the dependency.

Should I make a PR? Am I missing something here?

I'm on `ember-auto-import` version `1.5.1`

**Comment by Sticksword** [comment]
Running this fixes the issue, which further leads me to believe `@babel` or something got updated and now `ember-auto-import` is missing the dependency.
```
npm install --save-dev @babel/preset-env
```

**Comment by Sticksword** [comment]
Although... installing `@babel/preset-env` causes a slew of new problems. Not sure what's going on, will report back.

**Comment by Sticksword** [comment]
Since this [commit](https://github.com/ef4/ember-auto-import/commit/45ca1912713c2b178326134b837bc9ad9b8a5668)was recent, it makes me believe this is an actual bug.

**Comment by ef4** [comment]
You're right, this is a bug. Our linters would have caught this if we were trying to import from a package we forgot to put in package.json, but here we only `require.resolve` it to pass its path to babel.

Most likely this bug is only noticeable in apps that are still on babel 6.

**Comment by ef4** [comment]
Should be fixed in 1.5.2.
